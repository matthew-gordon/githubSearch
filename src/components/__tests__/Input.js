import React from 'react'
import { render } from '@testing-library/react-native'

import TextInput from '../common/Input'

describe('Input', () => {
  test('renders without crashing', async () => {
    const { baseElement } = render(<TextInput />)

    expect(baseElement).toMatchSnapshot()
  })
})
