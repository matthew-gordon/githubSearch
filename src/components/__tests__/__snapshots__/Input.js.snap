// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders without crashing 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      rejectResponderTermination={true}
      style={
        Array [
          Object {
            "backgroundColor": "#fff",
            "borderRadius": 10,
            "borderWidth": 1,
            "height": 42,
            "paddingLeft": 10,
            "paddingRight": 10,
            "width": "100%",
          },
        ]
      }
      underlineColorAndroid="transparent"
    />
  </View>
</View>
`;
